datasets/iowa_liquor_sales/docs/tutorials/sales_prediction/sales_prediction_test.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from testbook import testbook
+
+
+@pytest.mark.timeout(900)
+@testbook(
+    "datasets/iowa_liquor_sales/docs/tutorials/liquor_sales_prediction/liquor_sales_predictions.ipynb"
+)
+def test_run_notebook(tb):
+    tb.inject(
+        """
+        from unittest import mock
+        import pandas as pd
+        mock_client = mock.MagicMock()
+        mock_df = pd.DataFrame()
+        mock_df['week'] = range(50)
```

Comment:
You want to change mock_df. These lines are added, so our test code does not make an actual request to BigQuery. Instead, we "mock" the BQ call, and create a mock object (in this case, mock_df) to be the object we pretend that our BQ call returned. You want it to be a dataframe that you expect the BQ call returns, so the rest of your code can function and work properly.